cc_plugin_ugrid/checker_10.py

```diff
@@ -120,21 +121,69 @@ def check_node_coordinates_exist(self, ds):
             return None
         except AttributeError:
             # No node_dimensions attribute... there are larger issues
+            self.node_dim = None
             return None
         except AssertionError:
             m = '"node_coordinates" member "{}" is not a variable'.format(n)
             messages.append(m)
+            self.node_dim = None
+        else:
+            # verify that the node coordinates' dimensions are equivalent
+            node_coord = n
```

Comment:
what is `n` here? Is it the last iteration of above loop?

Reply:
@brianmckenna Yes, as the last value of the node_coordinates variable should be the same dimensions. Does that make sense?